website/versioned_docs/version-20.x/troubleshooting/building-the-app.md

```diff
@@ -128,3 +128,27 @@ packagingOptions {
     exclude 'META-INF/LICENSE'
 }
 ```
+
+### Running Detox in a Rosetta environment
+
+When working with dependencies that require running your iOS app in a Rosetta simulator, you may encounter issues with the `detox build` command. These issues often relate to SwiftEmitModule or SwiftCompile errors. To resolve this, follow these steps:
+
+1. Modify your build command in the Detox configuration:
+
+```json
+"build": "xcodebuild -workspace ios/MyApp.xcworkspace -scheme MyApp -configuration Debug -sdk iphonesimulator -arch x86_64 -derivedDataPath ios/build"
+```
+
+2. Run the following command in your terminal to ensure Xcode is properly selected:
+
+```bash
+sudo xcode-select --switch /Applications/Xcode.app/Contents/Developer
+```
```

Comment:
IMO we can assume they already have it configured. You only need to configure `xcode-select -s` when you have more than one version of Xcode installed on your system and you want to switch between them

Reply:
Let's remove this step since it's not unique for Rosetta env